torch/testing/_internal/common_utils.py

```diff
@@ -940,7 +881,9 @@ def read_and_process():
                     f"Test is disabled because an issue exists disabling it: {issue_url}" +
                     f" for {'all' if platforms == [] else ''}platform(s) {', '.join(platforms)}." +
                     " To enable, set the environment variable PYTORCH_RUN_DISABLED_TESTS=1")
-
+    if TEST_SKIP_FAST:
+        if not getattr(test, test._testMethodName).__dict__.get('slow_test', False):
+            raise unittest.SkipTest("test is fast; we disabled it with PYTORCH_TEST_SKIP_FAST")
```

Comment:
I still do think this should be grouped with lines 866~, but can be done later

Reply:
no exactly. slow_test can be marked with `@slowTest` wrapper from common_utils.py as well. so the test_name might not be a key in the `slow_tests_dicts`

Reply:
I mean like this code itself should come before the disabled stuff

Reply:
ah. that make sense actually . will incorporate in the next batch of refactoring